audit/projects/k8s-infra-e2e-boskos-gpu-02/buckets/kubernetes-staging-2abfb0e14f/bucketpolicyonly.txt

```diff
@@ -0,0 +1,3 @@
+Bucket Policy Only setting for gs://kubernetes-staging-2abfb0e14f:
+  Enabled: False
```

Comment:
nit for followup: I would like us to consider enabling `constraints/storage.uniformBucketLevelAccess` so kube-up doesn't create buckets without ubla enabled

ref: https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints